src/wp-admin/includes/template.php

```diff
@@ -1204,7 +1210,7 @@ function _get_plugin_from_callback( $callback ) {
 	}
 
 	// Don't show an error if it's an internal PHP function.
-	if ( ! $reflection->isInternal() ) {
+	if ( ! $reflection->isInternal() && false !== $reflection->getFileName() ) {
```

Comment:
The `false !== $reflection->getFileName()` covers internal and is a good check.  Checking the `isInternal` is not for the conditional check but it may be a more performant to check it first (maybe). See PHP src https://github.com/php/php-src/blob/master/ext/reflection/php_reflection.c to determine if more performant.

The comment needs refinement.